backend/heroes.py

```diff
@@ -299,3 +307,27 @@ def move(self, state):
           0, min(self.speed, distance + self.target.speed - 1))
         self.x += direction_x * step_size
         self.y += direction_y * step_size
+
+class Scientist(Hero):
+  name = 'Derek'
+  title = 'Head of Thoughtworm Research'
+  shape = shapes.scientist
+  in_conversation_with = None
+  influence_per_level = 0.2
+
+  abilities = [
+    {
+      'name': 'Teacher',
+      'description':
+      '''Derek gains inspiration from teaching. Every time he attacks a different
+opponent, his inspiration increases.''',
+      'unlockLevel': 1,
+    },
+    {
+      'name': "Aumann's agreement theorem",
+      'description':
+      '''When Derek meets another rationalist, they argue until one of them changes his oppinion.'''
```

Comment:
```suggestion
      '''When Derek meets another rationalist, they argue until one of them changes his opinion.'''
```

I've been thinking about phrasing these abilities. I think I also went too heavy on style to the detriment of explaining what actually happens. Our visual indicators are way too weak to hope that the player will see what happens. This text is all they have to go on. I think we must include a clear technical description.